data-host/library/src/popup/components/About.tsx

```diff
@@ -0,0 +1,78 @@
+
+import Button from "@common/Button";
+import Icon, { IconNames } from "@common/Icon";
+import Link from "@common/Link";
+import withNavBar from "@common/NavBar";
+import Text from "@common/Text";
+import Title from "@common/Title";
+import TopComponent from "@common/TopComponent";
+import styled from "styled-components";
+
+const ContentContainer = styled.div`
+  margin-top: var(--s-24);
+  margin-bottom: var(--s-10);
+`;
+const Version = styled.span``;
+const IconsContainer = styled.div`
+  display: flex;
+  justify-content: space-evenly;
+`;
+const ActionContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: var(--s-24) 0;
+`;
+
+type AboutProps = {
+  version: string;
+  onNext: () => void;
+  onClickFractalLink: () => void;
+  onClickFractalTelegram: () => void;
+};
+
+function About(props: AboutProps) {
+  const { onNext, onClickFractalLink, onClickFractalTelegram, version } = props;
+
+  return (
+    <TopComponent>
+      <ContentContainer>
+        <Title>
+          Fractal Wallet{" "}
+          <Version>
+            <Text>{`v${version}`}</Text>
+          </Version>
+        </Title>
+        <Text>
+          <br />
+        </Text>
```

Comment:
Is this necessary? If this is for spacing, please use actual CSS instead.

Reply:
Same elsewhere.

Reply:
I'd assume that `<Text>` has attached CSS?

Reply:
Probably, but my point is we want to use margins or something to space things. I'm not aware of any legitimate use of `<br />`, except for doing spacing that should be done with CSS.